qldbshell/__main__.py

```diff
@@ -59,6 +55,12 @@ def main():
         help="Name of a profile speficified in aws credentials setup whose credentials we should use",
         action="store",
     )
+    parser.add_argument(
+        "-q",
+        "--query-stats",
+        help="Enable display of query statistics",
+        action="store_true"
```

Comment:
Just to confirm, we want by default, the query stats enabled?

Reply:
They're disabled by default, and only enabled if you add the flag